docs/docs/build/examples/btc_swap/index.md

```diff
@@ -3,32 +3,143 @@
 This example demonstrates how BOB can be used to build a peer to peer swap application which allows BTC to be exchanged for an ERC20 token without the use of a custodian.
 
 :::note
+This example app is a work in progress, and the application repository is currently set to private.
```

Comment:
Let's make it public to allow people to see what's happening behind the scenes.

Reply:
We're going to rename the repo as soon as we have these last changes merged—as soon as that's done I'll make it public and add a link.